src/timers.c

```diff
@@ -33,7 +33,7 @@ extern volatile unsigned int recvd_signal_flags;
 struct timer {
   struct timer *next, *prev;
 
-  long count;                   /* Amount of time remaining */
+  long remaining;               /* Amount of time remaining */
```

Comment:
To be even more descriptive, would you be open to making this be `remaining_secs`?

Reply:
I'm not totally opposed to that.  Then would you want to also rename `interval` to `interval_secs`, and the `elapsed` variable in process_timers() to `elapsed_secs` for consistency?

Reply:
Yes, that would be great.  Thanks!